ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButton: UIButton {
+    let item: CommandBarItem
+
+    override var isHighlighted: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    override var isSelected: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    override var isEnabled: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    init(item: CommandBarItem, isFixed: Bool = false) {
```

Comment:
"isFixed" the right property name? from commandbarbutton point of view it doesn't make much sense.  ithink it just translated to whether or not this button isSelectable?

Reply:
Renamed to `isPersistSelection`.